README.md

```diff
@@ -391,19 +405,61 @@ is provided.
 
 
 ## HTTP status code handler
-Http Sink and Lookup Source connectors allow defining list of HTTP status codes that should be treated as errors. 
+### Sink table
+Http Sink allows defining list of HTTP status codes that should be treated as errors. 
```

Comment:
nit: 
maybe 
You can configure a list of HTTP status codes that should be treated as errors for HTTP sink table